packages/calling/src/Voicemail/types.ts

```diff
@@ -70,22 +70,141 @@ export type VoicemailResponseEvent = {
   message: string | null;
 };
 
+/**
+ * Represents an interface for managing voicemail-related operations.
+ */
 export interface IVoicemail {
-  getSDKConnector: () => ISDKConnector;
-  init: () => VoicemailResponseEvent;
-  getVoicemailList: (
+  /**
+   * Retrieves the SDK connector associated with the Calling SDK.
+   *
+   * @returns The SDK connector as {@link ISDKConnector}.
+   */
+  getSDKConnector(): ISDKConnector;
+
+  /**
+   * Initializes the voicemail service and returns a voicemail response event.
+   *
+   * @returns A voicemail response event as {@link VoicemailResponseEvent}.
```

Comment:
We are saying VoicemailResponse event is returned in every API but we don't cover what is being returned inside it as the response would be different for each API